components/datadog/agent/install.go

```diff
@@ -201,10 +201,9 @@ func writeFileDefinition(
 	lastCommand *remote.Command) (*remote.Command, error) {
 
 	var err error
-	folderPath, _ := path.Split(fullPath)
 
 	// create directory, if it does not exist
-	lastCommand, err = fileManager.CreateDirectory(folderPath, useSudo, utils.PulumiDependsOn(lastCommand))
+	lastCommand, err = fileManager.CreateDirectory(fullPath, useSudo, utils.PulumiDependsOn(lastCommand))
```

Comment:
```suggestion
	lastCommand, err = fileManager.CreateDirectoryForFile(fullPath, useSudo, utils.PulumiDependsOn(lastCommand))
```

Reply:
fixed in https://github.com/DataDog/test-infra-definitions/pull/399/commits/74520197adec21523407119c74cfd3e0c7394b24